experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -166,7 +166,7 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 			localSessionId,
 			undoRedoManager,
 			undoRedoManager.clone(() => this.editManager.getTrunkHead()),
-			anchors,
+			new AnchorSet(), // This class handles the anchor rebasing, so we don't want the editor to do any rebasing; so pass it a dummy anchor set.
```

Comment:
EditManager rebases anchors in three places:
1. When adding a local change
2. When adding a sequenced change (because local branch gets rebased over the new trunk head)
3. When aborting a transaction and "rolling back"

We want to avoid doing 1, but we still want to do 2 and 3. I say, revert this line here and just delete the three lines inside EditManager that do 1. Still sad that no tests broke... I guess I need to add more tests.